src/components/home/modals/Receive.vue

```diff
@@ -72,8 +73,13 @@
                          v-model="invoice"
                          :rows="3"
                          :no-resize="true"
+                         aria-describedby="inputInvoiceLiveFeedback"
                          :state="isValidInvoice">
         </b-form-textarea>
+        <b-form-invalid-feedback id="inputInvoiceLiveFeedback"
+                                 v-if="!isValidInvoice">
+          Invalid invoice. It may exceed the 140-character limit.
```

Comment:
update words to `The length of invoice is too long. It exceeds the max limit of 140.`